mix.exs

```diff
@@ -36,56 +36,60 @@ defmodule Membrane.Gigachad.Mixfile do
+      {:membrane_aac_fdk_plugin, "~> 0.17.0"},
       {:membrane_realtimer_plugin, "~> 0.6.1"},
-      {:membrane_framerate_converter_plugin, "~> 0.6.0"},
-      # {:membrane_video_compositor_plugin, "~> 0.2.1"},
-      {:membrane_generator_plugin, "~> 0.8.0"},
-      # {:membrane_rtc_engine, "~> 0.11.0"},
-      {:membrane_audio_mix_plugin, "~> 0.12.0"},
-      {:membrane_audio_filler_plugin, "~> 0.1.0"},
-      {:membrane_rtmp_plugin, "~> 0.11.2"},
-      {:membrane_flv_plugin, "~> 0.5.0"},
-      {:membrane_ffmpeg_swscale_plugin, "~> 0.11.0"},
-      {:membrane_video_merger_plugin, "~> 0.7.0"},
-      {:membrane_stream_plugin, "~> 0.2.1"},
-      # {:membrane_sdl_plugin, "~> 0.15.0"},
-      {:membrane_ffmpeg_video_filter_plugin, "~> 0.9.0"},
-      {:membrane_ffmpeg_swresample_plugin, "~> 0.16.1"},
-      {:membrane_udp_plugin, "~> 0.9.2"},
-      {:membrane_h264_plugin, "~> 0.2.1"},
-      {:membrane_portaudio_plugin, "~> 0.15.0"},
+      {:membrane_framerate_converter_plugin, "~> 0.7.0"},
+      {:membrane_generator_plugin, "~> 0.9.0"},
+      {:membrane_rtc_engine, "~> 0.17.0"},
+      {:membrane_audio_mix_plugin, "~> 0.15.0"},
+      {:membrane_audio_filler_plugin, "~> 0.2.0"},
+      {:membrane_rtmp_plugin, "~> 0.17.2"},
+      {:membrane_flv_plugin, "~> 0.9.0"},
+      {:membrane_ffmpeg_swscale_plugin, "~> 0.14.0"},
+      {:membrane_video_merger_plugin, "~> 0.8.0"},
+      {:membrane_stream_plugin, "~> 0.3.1"},
+      {:membrane_sdl_plugin, "~> 0.17.0"},
+      {:membrane_ffmpeg_video_filter_plugin, "~> 0.11.0"},
+      {:membrane_ffmpeg_swresample_plugin, "~> 0.18.0"},
+      {:membrane_udp_plugin, "~> 0.10.0"},
+      {:membrane_h264_plugin, "~> 0.7.0"},
+      {:membrane_portaudio_plugin, "~> 0.17.0"},
       {:bunch, "~> 1.6", override: true},
-      {:membrane_mp3_mad_plugin, "~> 0.14.0"},
-      {:membrane_fake_plugin, "~> 0.9.0"},
-      {:membrane_hackney_plugin, "~> 0.9.0"},
-      {:membrane_raw_video_parser_plugin, "~> 0.9.0"},
-      {:membrane_wav_plugin, "~> 0.8.0"},
-      {:membrane_rtp_mpegaudio_plugin, "~> 0.12.0"},
-      {:membrane_audiometer_plugin, "~> 0.10.1"},
-      {:membrane_camera_capture_plugin, "~> 0.4.0"},
-      {:membrane_ivf_plugin, "~> 0.5.0"},
-      {:membrane_matroska_plugin, "~> 0.2.1"},
-      {:membrane_scissors_plugin, "~> 0.6.0"},
-      {:membrane_flac_plugin, "~> 0.9.0"},
-      {:membrane_ffmpeg_generator, "~> 0.1.0"},
-      {:membrane_ogg_plugin, "~> 0.1.0"},
-      {:membrane_mp3_lame_plugin, "~> 0.15.0"},
+      {:membrane_mp3_mad_plugin, "~> 0.17.0"},
+      {:membrane_fake_plugin, "~> 0.10.0"},
+      {:membrane_hackney_plugin, "~> 0.10.0"},
+      {:membrane_raw_video_parser_plugin, "~> 0.11.2"},
+      {:membrane_wav_plugin, "~> 0.9.2"},
+      {:membrane_rtp_mpegaudio_plugin, "~> 0.13.0"},
+      {:membrane_audiometer_plugin, "~> 0.11.0"},
+      {:membrane_raw_audio_format, "~> 0.11.0"},
+      {:membrane_camera_capture_plugin, "~> 0.6.0"},
```

Comment:
this has some problems with the precompiled FFmpeg, @Noarkhh is working on that

Reply:
removed